src/Conductors/ChecksRole.php

```diff
@@ -47,6 +47,17 @@ public function a($role)
     }
 
     /**
+     * Check if the user doesn't have any of the given roles.
+     *
+     * @param  string  $role
+     * @return bool
+     */
+    public function notA($role)
```

Comment:
`func_get_args()` can only get the arguments of that function, not of its caller.

The reason that test passes is because it's not a good test; it passes the existing role first, so it passes even if the other roles aren't checked. I've now changed that test: https://github.com/JosephSilber/bouncer/commit/bd83a08890b064f2825894df2a3a8826a812a8c8

Rebase your code and you'll see that the tests will fail.